### ISSUE TITLE
Call of `getDocuments()` with empty document ID list returns all documents

### ISSUE THREAD

**Issue by ChristophB** [Onto-Med/top-backend#219]
For queries with an empty result set, `getDocuments(...)` is called with an empty array as `documentIds` parameter and returns all documents. The expected behaviour is to return an empty list of documents.

**Comment by fmatthies** [comment]
That's right. This is an oversight in the interaction between `frontend` ↔ `backend`.
However, I would rather intercept this case in the `frontend`.

**Comment by ChristophB** [comment]
I would prefer to handle this case in the backend, because it does not feel right from an REST API perspective to return all documents.

**Comment by fmatthies** [comment]
I understand the notion, but I have the feeling, that it's kind of odd to provide an empty filter (if it's empty nothing gets filtered) and get nothing in return. You might as well not request it at all if you know it gives an empty return?
